github-linguist.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'charlock_holmes', '~> 0.7.3'
   s.add_dependency 'escape_utils',    '~> 1.1.0'
   s.add_dependency 'mime-types',      '>= 1.19'
-  s.add_dependency 'rugged',          '0.26.0b3'
```

Comment:
The loosening from 0.2.6.0b3 was just me reversing the bump I did in https://github.com/github/linguist/pull/3473/commits/0b9eb0c1033a8af1afef35bc9d8f8cae1b113a11 (in this PR) and instead opting for a loosening of the version.  The overall change in this PR is from `= 0.25.1.1` to `>= 0.25.1.1`.